src/compress/gzip/header.cr

```diff
@@ -9,6 +9,7 @@ class Compress::Gzip::Header
   # :nodoc:
   @[Flags]
   enum Flg : UInt8
+    None    = 0
```

Comment:
I'd drop all the additions of `None = 0` from this PR, it's kinda getting ahead of ourselves.